pkg/apis/sources/v1beta1/implements_test.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package v1beta1
+
+import (
+	"testing"
+
+	"knative.dev/pkg/apis/duck"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+	duckv1alpha1 "knative.dev/pkg/apis/duck/v1alpha1"
+)
+
+func TestTypesImplements(t *testing.T) {
+	testCases := []struct {
+		instance interface{}
+		iface    duck.Implementable
+	}{
+		// SinkBinding
+		{instance: &SinkBinding{}, iface: &duckv1.Conditions{}},
+		{instance: &SinkBinding{}, iface: &duckv1.Source{}},
+		{instance: &SinkBinding{}, iface: &duckv1alpha1.Binding{}},
```

Comment:
Can you file an issue (or pkg pr) to get this bumped to v1beta1 as well?

Reply:
Sure thing, will create a PR

Reply:
Done, here: https://github.com/knative/pkg/pull/1483